src/Exceptionless.Core/Mail/SmtpMailSender.cs

```diff
@@ -20,16 +21,27 @@ public class SmtpMailSender : IMailSender {
             message.Headers.Add("X-Auto-Response-Suppress", "All");
             message.Headers.Add("Auto-Submitted", "auto-generated");
 
-            var client = new SmtpClient();
-            if (!String.IsNullOrEmpty(Settings.Current.SmtpHost)) {
-                client.Host = Settings.Current.SmtpHost;
-                client.Port = Settings.Current.SmtpPort;
-                client.EnableSsl = Settings.Current.SmtpEnableSsl;
-                client.Credentials = new NetworkCredential(Settings.Current.SmtpUser, Settings.Current.SmtpPassword);
-            }
+            using (var client = new SmtpClient()) {
```

Comment:
Yes, I can't find any official remarks on this, my initial thoughts would've been that having a long running smtp connection for unobservable benefits is not worth the complexity.

I was unable to find anything in the [MailKit source](https://github.com/jstedfast/MailKit/blob/e8338c43b44d013b5384b16bbc46f532bccd8682/MailKit/Net/Smtp/SmtpClient.cs#L1660) that indicates it will try to re-connect if it becomes disconnected, so we would have to handle the case where it throws and re-connect manually.